book/layout.md

```diff
@@ -31,11 +31,11 @@ the HTML tree to produce the layout tree, then computes the size and position
 for each layout object, and finally draws each layout object to the screen.
 
 [^no-box]: Elements like `<script>` don't generate layout objects, and
-    some elements generate multiple (`<li>` elements have a layout
-    object for the bullet point!), but mostly it's one layout object
+    some elements generate multiple layout objects (`<li>` elements
+    have an extra one for the bullet point!), but mostly it's one layout object
```

Comment:
Here the copyeditor suggested "objects" in place of "layout objects" at the site of the first edit, and made no change to the second. But I think we shortening "layout objects" to "objects" is too confusing, and using "layout objects" necessitates changing the parenthetical to avoid annoying repetition.